src/modules/robot/Robot.cpp

```diff
@@ -1190,16 +1186,12 @@ bool Robot::append_arc(Gcode * gcode, const float target[], const float offset[]
     float millimeters_of_travel = hypotf(angular_travel * radius, fabsf(linear_travel));
 
     // discard segments so small they cause no movement
-    if ((gcode->has_letter('X')) && (millimeters_of_travel < (1/(actuators[0]->get_steps_per_mm())))) {
+    if (((gcode->has_letter('X')) && (millimeters_of_travel < (1/(actuators[0]->get_steps_per_mm())))) && 
+        ((gcode->has_letter('Y')) && (millimeters_of_travel < (1/(actuators[1]->get_steps_per_mm())))) &&
+        ((gcode->has_letter('Z')) && (millimeters_of_travel < (1/(actuators[2]->get_steps_per_mm()))))) {
```

Comment:
@wolfmanjm This formula still isn't right, and I know we don't need this.. but while I was putting it together I had the question of formatting for very long formulas like this.  

What is the preference for the smoothie project?  leave it one big long line, or break it up as I have here?  

Thanks for the advice